src/cli.rs

```diff
@@ -44,37 +41,38 @@ pub fn build_app<'a, 'b>() -> App<'a, 'b> {
             (@arg debug: --debug "Build without optimizations")
             (@arg profile: -p --profile default_value[local]
                 "Set testing profile. Run `oasis config profile` \nto list available profiles.")
-            (@group artifacts =>
-                (@arg SERVICE: +multiple "Specify which service(s) to build")
-                (@arg only_services: --services "Build all services (and no apps)")
-            )
+            (@arg TARGETS: +multiple "Specify names or paths of services and apps to build")
             (@arg tester_args: +raw "Args to pass to language-specific test tool")
         )
-        (@subcommand clean => (about: "Remove build products"))
-        (@subcommand chain =>
-            (about: "Run a local Oasis blockchain")
-            (@arg chain_args: +multiple "Args to pass to oasis-chain")
-        )
-        (@subcommand ifextract =>
-            (about: "Extract interface definition(s) from a service.wasm")
-            (@arg out_dir: -o --out +takes_value
-                "Where to write the interface.json(s). \
-                 Defaults to current directory. Pass `-` to write to stdout.")
-            (@arg SERVICE_PATH: +required "The path to service.wasm file")
-        )
         (@subcommand deploy =>
             (about: "Deploy services to the Oasis blockchain")
             (@arg verbose: +multiple -v --verbose "Increase verbosity")
             (@arg quiet: +multiple -q --quiet "Decrease verbosity")
             (@arg profile: -p --profile default_value[default]
                 "Set testing profile. Run `oasis config profile` \nto list available profiles.")
+            (@arg TARGETS: +multiple "Specify names or paths of services and apps to build")
             (@arg deployer_args: +raw "Args to pass to language-specific deployment tool")
         )
+        (@subcommand clean =>
+            (about: "Remove build products")
+            (@arg TARGETS: +multiple "Specify names or paths of services and apps to build")
```

Comment:
`Specify names or paths of services and apps to clean`?

Reply:
```suggestion
            (@arg TARGETS: +multiple "Specify names or paths of services and apps to clean")
```